app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,63 @@
+
+<section class="task-list">
+  <h3>your tasks</h3>
+  <ul>
+    <% @tasks.each do |task| %>
+      <li>
+        <% if task.status == true %>
+        <p>
+          <strong class="complete">
+          <%= task.title %>
+          </strong>
+        </p>
+        <div class="actions">
+          <span>
+            <%= link_to "mark incomplete", mark_as_incomplete_path(task.id), method: :patch, data: { confirm: "You sure you're not done this?" }, class: "link" %>
+          </span>
+          |
+          <span>
+            <%= link_to "edit", edit_task_path(task.id), class: "link" %>
+          </span>
+          |
+          <span>
+            <%= link_to "delete", tasks_path(task.id), class: "link" %>
```

Comment:
Looks like your delete link doesn't have the HTTP method set if the task is complete. Probably just a copy/paste error. Is there a way you could DRY up this code?